.eslintrc.yaml

```diff
@@ -0,0 +1,18 @@
+env:
+  es2020: true
+
+ignorePatterns:
+  - dist
+
+plugins:
+  - jest
```

Comment:
if you extend `plugin:@foxglove/jest` it will include this plugin for you + add recommended rules